2023/03.md

```diff
@@ -85,8 +85,9 @@ Supporting materials includes slides, a link to the proposal repository, a link
 
     | ✓ | stage | timebox | topic | presenter |
     |:-:|:-----:|:-------:|-------|-----------|
-    |   | 3 | 10m | Iterator helpers: validate arguments before opening iterator ([issue](https://github.com/tc39/proposal-iterator-helpers/issues/264), [PR](https://github.com/tc39/proposal-iterator-helpers/pull/265)) | Kevin Gibbons |
+    |   | 3 | 10m | [ADDED LATE] Iterator helpers: validate arguments before opening iterator ([issue](https://github.com/tc39/proposal-iterator-helpers/issues/264), [PR](https://github.com/tc39/proposal-iterator-helpers/pull/265)) | Kevin Gibbons |
     |   | 3 | 10m | Iterator helpers: closing iterators which have not been iterated ([issue](https://github.com/tc39/proposal-iterator-helpers/issues/266), [PR](https://github.com/tc39/proposal-iterator-helpers/pull/267)) | Kevin Gibbons |
+    |   | 3 | 20m | Iterator helpers: renaming `take` ([issue](https://github.com/tc39/proposal-iterator-helpers/issues/71#issuecomment-1461841551)) | Kevin Gibbons |
```

Comment:
did you put "added late" on the wrong item?

Reply:
Lol, yes, sorry.